SCClassLibrary/Common/GUI/tools/Stethoscope.sc

```diff
@@ -75,8 +75,17 @@ Stethoscope {
 		cycle = cycleSpec.constrain(cycle_);
 		yZoom = 1.0;
 
-		smallSize = Size(250,250);
-		largeSize = Size(500,500);
+		# smallSize, largeSize = if(parent.isNil) {
+			[
+				Size(250,250),
+				Size(500,500)
+			]
+		} {
+			[
+				Size(parent.bounds.width, parent.bounds.height),
+				Size(parent.bounds.width * 2, parent.bounds.height * 2)
+			]
+		};
```

Comment:
I did this to avoid typing the repeated words, but I corrected it according to your advice:
https://github.com/supercollider/supercollider/pull/6281/commits/a6bbbcf942dfe979d81b40d4ffbb6d7e267888c2